composer.json

```diff
@@ -8,9 +8,11 @@
     "doctrine/doctrine-migrations-bundle": "^1.3",
     "doctrine/orm": "^2.5.11",
     "eightpoints/guzzle-bundle": "^7.3",
+    "friendsofsymfony/ckeditor-bundle": "^2.1",
     "sensio/framework-extra-bundle": "^5.1",
     "sensiolabs/security-checker": "^5.0",
     "stof/doctrine-extensions-bundle": "^1.3",
+    "symfony/apache-pack": "^1.0",
```

Comment:
I worked without Docker...

Reply:
I think you should not add your personal configuration setup in an open source project just because you do not want use the default development environment. It’s up to you to use Docker or not, but it should not reflect in the PR.

Reply:
@guillaumebriday I agree completely. 

Now I see that you have a Nginx config file within Docker. Will remove this dependency off course.